brain-marks/ContentView.swift

```diff
@@ -22,6 +22,14 @@ struct ContentView: View {
                     Text("Settings")
                 }
         }
+        .onOpenURL(perform: { url in
```

Comment:
This video may help!
https://youtu.be/H-MNG-GS8J0

Reply:
Was watching this today, definitely helpful! I figured if I could not get the sample widget working from the video then it might be an Apple bug. You'll be shocked to know: it's not an Apple bug.